README.rst

```diff
@@ -19,6 +19,8 @@ Hopefully this is enough to get you going::
     -v, --verbose
     -d, --dump
     -u UUID, --uuid=UUID  Camera UUID
+    -i id, --id=id        Camera Connection id
+    -c, --connect_with_id Connect using id instead of UUID (NVR Beta API)
```

Comment:
The api path is the same... 2.0 I can try to pull more data out of the api but I am just capturing data with chrome since the api is not documented.